### DISCUSSION TITLE
does liquidctl suppoert aorus waterforce x240

### DISCUSSION THREAD

**Discussion by coding404life** [liquidctl/liquidctl#578] [General]
Hello everyone my liquid cooler aorus waterforce x240 seems not to be supported by liquidctl

**Comment by coding404life** [comment]
i hope anyone can help me here

**Comment by amazonparrot** [comment]
Please check this thread, I think they are working on it, I am using Waterforce x240 too. https://github.com/liquidctl/liquidctl/issues/167
